src/Bank/BankLogic/AccountType.cs

```diff
@@ -1,8 +1,8 @@
-namespace BankingSystem.Bank
+namespace BankingSystem.BankLogic
 {
     public enum AccountType
     {
-        CheckingAccount,
-        SavingsAccount,
+        CheckingAccount = 1,
```

Comment:
1, 2 it by default

Reply:
It's because the default value start on "0" and I thought that choose  1 is more natural for human, for example in this case:
```
Specify the 'type' of the account
        1 (Checking account)
        2 (Saving account)
```